pkgs/by-name/qt/qt-scrcpy/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libsForQt5,
+  scrcpy,
+}:
+stdenv.mkDerivation rec {
+  pname = "qt-scrcpy";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "barry-ran";
+    repo = "QtScrcpy";
+    rev = "v${version}";
+    hash = "sha256-PL/UvRNqvLaFuvSHbkJsaJ2nqRp5+ERM+rmlKVtbShk=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = with libsForQt5; [
+    qtbase
+    wrapQtAppsHook
+    qtmultimedia
+    scrcpy
+    qtx11extras
+  ];
+
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-Wno-error=sign-compare"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp -r ../output/x64/Release/* $out/bin
```

Comment:
Doesn't this automatically happen?

Reply:
I suppose it should be if we are not overriding `installPhase`. Also `runHook postInstall` is missing here.

Reply:
I think it should happen automatically, but it seems there is no make install target in the `CmakeLists.txt` file, so the manual route works